packages/orbit-components/transforms/Modal-size.js

```diff
@@ -0,0 +1,16 @@
+// @noflow
+/* eslint-disable no-param-reassign */
+function transformModalSizeProp(fileInfo, api) {
+  const j = api.jscodeshift;
+  const root = j(fileInfo.source);
+
+  return root
+    .find(j.JSXOpeningElement, { name: { name: "Modal" } })
+    .find(j.JSXAttribute, { name: { name: "size" }, value: { value: "large" } })
+    .forEach(path => {
+      path.node.value.value = "extraLarge";
+    })
```

Comment:
This is the happy path, but for example in `frontend/frontend` I found an instance with `size={size}`, so in cases where the value is not a string literal I think the codemod should output a warning like: "Unable to transform Modal size value because it's not a string literal, please verify whether `large` size is being used and use `extraLarge` instead.", along with the absolute file path with location information (line and column number) in a format that can be opened with Cmd+click in VS Code. You can extract line and column number from the AST (AST Explorer hides location data by default, but you can uncheck the checkbox above.)

For example:

```suggestion
    .find(j.JSXAttribute, { name: { name: "size" } })
    .forEach(path => {
      if (path.node.value.type === j.Literal) {
        path.node.value.value = "extraLarge";
      } else {
        // output warning
      }
    })
```

Reply:
it will not work, because `j.Literal` is an object. `j.Literal.name` will work

Reply:
but anyway, thanks for suggestion